frontend/portals/CartItemListAfter/index.jsx

```diff
@@ -5,7 +5,7 @@ import getConfig from '../../helpers/getConfig';
 export default () => {
   const config = getConfig();
 
-  if (config.checkboxValues.length === 0) {
+  if (!config.checkboxValues || config.checkboxValues.length === 0) {
```

Comment:
Why this change is needed?

Reply:
Leftover implementation for a test. Removed because I mock config in test now.

Reply:
Also removed unit test where I mocked null config.